From a04ffc9adaf16ae105ccd621e4365eb3008e3458 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@evolve-os.com>
Date: Sun, 25 Jan 2015 20:21:44 +0000
Subject: [PATCH 1/4] You print, you die.

---
 src/evoassist-ui.py | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/src/evoassist-ui.py b/src/evoassist-ui.py
index 06a0ad6..70fd93a 100644
--- a/src/evoassist-ui.py
+++ b/src/evoassist-ui.py
@@ -187,12 +187,13 @@ users!"""
         layout.pack_start(bugs, False, False, 10)
 
         return layout
-    def on_repl(self, o):            
-        print "ok %s" % str(o)
+    def on_repl(self, o):      
+        pass      
+        #print "ok %s" % str(o)
 
     def on_err(self, o):
         self.install_btn.set_sensitive(True)
-        print "error %s" % str(o)
+        #print "error %s" % str(o)
 
     def do_prog(self, pct, message):
         if str(message).startswith("ERROR: "):
@@ -213,7 +214,7 @@ users!"""
                 self.install_btn.hide()
                 self.tick.show()
         # for the curious/debugs
-        print message
+        #print message
 
 if __name__ == "__main__":
     DBusGMainLoop(set_as_default=True)
-- 
2.3.2

