From 0bfaaf713a628c70fa4992839617382802d28170 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Tue, 21 Apr 2015 06:32:14 +0100
Subject: [PATCH 4/4] Branding step #2

---
 src/evoassist-ui.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/evoassist-ui.py b/src/evoassist-ui.py
index c4a61d5..7f12f6c 100644
--- a/src/evoassist-ui.py
+++ b/src/evoassist-ui.py
@@ -82,8 +82,8 @@ class EvoWelcome(Gat.SidebarWindow):
     def create_welcome_page(self):
         layout = Gtk.VBox(0)
 
-        version = "Evolve OS"
-        with open("/etc/evolveos-release", "r") as inp:
+        version = "Solus Operating System"
+        with open("/etc/solus-release", "r") as inp:
             lines = inp.readlines()
             version = lines[0].replace("\r","").replace("\n", "")
 
-- 
2.3.2

